src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -156,9 +162,12 @@ class JesBackend extends Backend with LazyLogging {
     val engineFunctions = new JesEngineFunctions(GoogleCloudStoragePath(callGcsPath), JesConnection)
 
     // FIXME: Not particularly robust at the moment.
```

Comment:
I forget exactly what led me to put this comment here (outside of the obvious fact that it was true!) but would you say that it still applies? If not, remove it. If so, can you see if a little short term poking here could save longer term pain down the road?